frontend/www/js/omegaup/arena/submissions.ts

```diff
@@ -16,6 +20,18 @@ interface RunSubmit {
   runs: types.Run[];
 }
 
+interface SubmissionRequestWithExtraData {
```

Comment:
```suggestion
interface SubmissionResponse {
```

esto no es una petición, es la respuesta!

Reply:
Listo.